ephios/core/templates/core/event_list.html

```diff
@@ -106,35 +107,130 @@ <h1>{% translate "Events" %}
     {% if mode == "calendar" %}
         {{ calendar }}
     {% elif mode == "day" %}
-        <table class="table">
-            <thead>
-                <tr>
-                    <td>Time</td>
-                    {% for event in events %}
-                        <td>{{ event.title }}</td>
+        <style nonce="rAnd0m">
```

Comment:
pls move to CSS file for proper CSP stuff

Reply:
Hmm, how would I do this as I need to generate CSS dynamically? The Django csp plugin also has support for adding nonces (maybe also hashes) to style blocks, but I haven't looked into that yet.

Reply:
This should be handled correctly now.